### ISSUE TITLE
Have a look at ReLaXed for some pointers

### ISSUE THREAD

**Issue by mfrw** [ILUGD/splatter#1]
https://github.com/RelaxedJS/ReLaXed
https://github.com/RelaxedJS/ReLaXed-examples/tree/master/examples/poster/

**Comment by hellozee** [comment]
Thanks @mfrw , that will be really helpful

**Comment by Boot-Error** [comment]
Using Relaxed would have been a better idea since we can use HTML, CSS to design posters! Current implementation is a more like writing text over a background image, former one is good but beefy latter one is simple.

**Comment by hellozee** [comment]
@Boot-Error sure but that is what we have been doing using Canva, the problem I am trying to solve is not requiring human intervention to draw posters, as you can see the program is still in its infancy and I plan to include a template engine #2 in which the program will generate templates from XML and use those templates to create posters.

**Comment by Boot-Error** [comment]
Yeah I thought of the same, build a small template engine for conveniently theme the posters. We need to start with this then #3. I have some ideas for this, will let you know about the progress as a PR

**Comment by hellozee** [comment]
We can discuss your ideas on the #2 thread. And if you have taken some issue please leave a comment regarding that.
